src/walkthroughs/unit-tests/index.rst

```diff
@@ -7,28 +7,121 @@ Walkthrough: Unit Tests
-* Instructor will walk everyone through setting up the project
-* ``CarTest.java`` add this import line ``import static org.junit.Assert.*;``
-* Start completing the ``TODO`` comments with the instructor
-
-  a. Open the ``TODO`` window by
-  b. Going to the **View** menu
-  c. Then **Tool Windows**
-  d. Then click **TODO**
-  e. Intellij will show you all the files that contain the text ``TODO``
+* Create a new branch for the new unit tests we will be writing ``$ git checkout -b my-unit-test-solution``
+* Open IntelliJ
+* Choose **open folder** and navigate to and choose folder **car-unit-test-starter**
+
+
+Configure IntelliJ for Java 8
+-----------------------------
+
+Before we can write any unit tests we need to configure this project for Java 8.
+
+Check out the `Configure IntelliJ <../../configurations/intellij/>`_ article for detailed steps.
+
+As a reminder from that configuration article you will need to 
+  * Set our Project SDK
+  * Set our Project language level
+  * Set our Project compiler output
+  * Mark the main directory as Source Root
+  * Mark the test directory as Test Sources Root
+
+Add an Empty Test
+-----------------
+
+We need to create an empty test so that we can create our testing runtime configuration. However, IntelliJ will only find our test class if it has a valid test in the class body.
+
+* In CarTest.java add an empty test
+
+.. image:: /_static/images/unit-tests/empty-test.png
+
+* Add JUnit 4 to your class path (click @Test and then option+enter)
+
+.. image:: /_static/images/unit-tests/junit4-classpath.png
+
+* Import Test, and AssertEquals
+
+.. image:: /_static/images/unit-tests/import-assertequals.png
+
+Create a New Runtime Configuration
+---------------------------------- 
+
+Checkout the `runtime configuration <../../configurations/runtime-configuration/>`_ guide for a step-by-step guide for getting your runtime configuration created
+
+As a reminder from the runtime configuration article you will need to
+  * Add a new configuration
+  * Name the configuration
+  * Select the class file for our tests
+  * Apply the configuration
+  * Run new configuration by clicking the green arrow
+  * View test results 
+
+.. image:: /_static/images/unit-tests/first-test-results.png
+
+Currently our one test passes, because emptyTest is our only test, and it's checking if 10 is 10, which is true.
+
+Write and Run our Unit Tests
+----------------------------
+* Write tests
+
+.. image:: /_static/images/unit-tests/completed-tests.png
+
+Our first three tests are very similar to one another. They each have a test_car object, and are checking that the gas tank level is correct after instantiating the car, driving the car within the range, and driving the car exceeding the range.
+
+The assertEquals() method takes three parameters: The value we want to test, What it's value should be, and the delta (how much the actual, and expected values can differ and still return true).
```

Comment:
IMHO functions and code like **assertEquals()** should be inside an inline code style  ``assertEquals()`` by wrapping it with double ` on each side

Reply:
Methods have been wrapped by double 1.